### DISCUSSION TITLE
Error: no vendor directory

### DISCUSSION THREAD

**Discussion by SESA634269** [brefphp/bref#1246] [General]
**Requesting help needed Php lambda logs showing error from runtime bootstrap file:**

PHP Warning:  require(/var/task/vendor/autoload.php): Failed to open stream: No such file or directory in /opt/bootstrap on line 25

PHP Fatal error:  Uncaught Error: Failed opening required '/var/task/vendor/autoload.php' (include_path='.:/opt/bref/lib/php') in /opt/bootstrap:25

**How to reproduce:**
Actually We where using PHP 7.3 FPM layer - arn:aws:lambda:eu-west-1:209497400698:layer:php-73-fpm:25
but because of Invoke error happing very frequently we decided to update Bref library for PHP 8.0 so following this repo -https://github.com/aws-samples/php-examples-for-aws-lambda/tree/master/0.1-SimplePhpFunction#creating-dependencies, created new vendor with bref  for Php 8.0 installed with in cloud9. but getting above mentioned error with layer - arn:aws:lambda:eu-west-1:209497400698:layer:php-80-fpm:40 , from runtime bref. because the original php lambda is not developed by me also i do not have such deep knowledge in Php I'm really seeking for some help around.

This is the file structure of original working PHP lambda , I'm also trying to replicate the same structure. so As shown in below we have vendor folder next to main source code and lambda have only one layer provided by bref runtime. 
<img width="86" alt="image" src="https://user-images.githubusercontent.com/90765436/178727675-b21a8762-500e-40d6-806f-fe06cb9861e0.png">

please let me know any solution Thanking in advance.

**Comment by mnapoli** [comment]
The zip that is deployed to Lambda probably doesn't contain the `vendor/` directory. You should try downloading the zip file in the AWS Console, and see if everything looks good inside.

**Reply by SESA634269** [reply]
Please see this <<
require 'vendor/autoload.php';
use Aws\S3\S3Client;
use Aws\S3\Exception\S3Exception;
use Aws\Ses\SesClient;
use Aws\Exception\AwsException;
use Bref\Context\Context;
include 'objects/input.php';>>

 this lines is for importing . is same works for layer only- arn:aws:lambda:eu-west-1:209497400698:layer:php-73-fpm:25 and not for any other version.
